[OSDOCS-4294]: adding new mce overview topic

lahinson opened the pull request:
<!--- PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][OCPBUGS#<jira-issue-id>][OSDOCS-<jira-issue-id>]: <short-description-of-the-pr> --->

<!--- If your changes apply to the latest release and/or in-development version of OpenShift, open your PR against the `main` branch. --->

Version(s): 4.12+
<!--- Specify the version or versions of OpenShift your PR applies to. -->

<!-- Version examples:
  * PR applies to all versions after a specific version (e.g. 4.8): 4.8+
  * PR applies to the in-development version (e.g. 4.12) and future versions: 4.12+
  * PR applies only to a specific single version (e.g. 4.10): 4.10
  * PR applies to multiple specific versions (e.g. 4.8-4.10): 4.8, 4.9, 4.10 --->

Issue: https://issues.redhat.com/browse/OSDOCS-4294
<!--- Add a link to the Bugzilla, Jira, or GitHub issue, if applicable. --->

Link to docs preview: https://52801--docspreview.netlify.app/openshift-enterprise/latest/architecture/mce-overview-ocp.html
<!--- Add direct link(s) to the exact page(s) with updated content from the preview build. --->

QE review:
- [x] QE has approved this change.
<!--- QE approval is required to merge a PR except for changes that do not impact the meaning of the docs. --->

This has been reviewed by several SMEs, including the ACM/MCE PM, the HyperShift PM, and the ACM/MCE documentation manager.

<!-- NOTE:
Automatic preview functionality is currently only available for some branches. For PRs that update the rendered build in any way against branches that do not create an automated preview:
  * OpenShift documentation team members (core and aligned) must include a link to a locally generated preview.
  * External contributors can request a generated preview from the OpenShift documentation team. --->

Additional information:
<!--- Optional: Include additional context or expand the description here.--->

<!--- Next steps after opening your PR:

* Ask for review from the OpenShift docs team:
  - For community authors: Request a review by tagging @openshift/team-documentation in a GitHub comment.
  - For Red Hat associates:
    * For normal peer requests, add a comment that contains this text: /label peer-review-needed
    * For normal merge review requests, add a comment that contains this text: /label merge-review-needed
    * For urgent peer review requests, ping @peer-review-squad requesting a review in the #forum-docs-review channel (CoreOS Slack workspace) and provide the following information:
      * A link to the PR.
      * The size of the PR that the GitHub bot assigns (ex: XS, S, M, L, XL).
      * Details about how the PR is urgent.
    * For urgent merge requests, ping @merge-review-squad in the #forum-docs-review channel (CoreOS Slack workspace).
    * Except for changes that do not impact the meaning of the content, QE review is required before content is merged.

* IMPORTANT:
  - All documentation changes must be verified by a QE team associate before merging.
  - Squash to one commit before submitting your PR for peer review.

* For more information about verifying your content, see:
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#verification-of-your-content

* For more information about contributing to OpenShift documentation, see:
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/contributing.adoc

Additional resources

The OpenShift docs repo adheres to the following style guides:

- OpenShift documentation guidelines (OSDOCS)
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc
- Red Hat Supplementary Style Guide (SSG)
  https://redhat-documentation.github.io/supplementary-style-guide/
- Modular Documentation Reference Guide (Mod Docs)
  https://redhat-documentation.github.io/modular-docs/
- IBM Style Guide (ISG)
  https://www.ibm.com/docs/en/ibm-style

You can log in to the ISG by using your @redhat.com id and single sign-on (SSO) credentials. --->

zanetworker commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,28 @@
+* Zero touch provisioning (ZTP), which provides full automation for cluster creation on bare metal and vSphere with minimal intervention. ZTP also offers a GitOps workflow for managing deployments.
+* Support for hosted control planes for {product-title}.
+
+The operator is entitled with a Red Hat OpenShift support subscription and is delivered separately from the core OpenShift payload. To start to use mulicluster engine, you must deploy the OpenShift cluster and then install the operator. 
```
Can we point to the install docs here?

lahinson replied:
Sure! I'll add the link now.

zanetworker reviewed (commented):
LGTM overall, but needs more review from ACM PM as well as CIM/ZTP PM.

berenss commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,28 @@
+One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator. The operator delivers full lifecycle capabilities for managed OpenShift clusters and partial lifecycle management for other Kubernetes distributions. It is available as part of Red Hat Advanced Cluster Management fo ...
+
+If you use Red Hat Advanced Cluster Management, multicluster engine is included and provides the following capabilities:
+
```
I would make 2 sentences:
"If you use Red Hat Advanced Cluster Management, the multicluster engine is already included as a prerequisite. 
The multicluster engine provides the following capabilities:"

julienlim replied:
Can we add a hyperlink to RHACM for the RHACM mention?

berenss commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,28 @@
+
+* Hive, which provisions and manages self-managed OpenShift clusters to the hub.
+* Zero touch provisioning (ZTP), which provides full automation for cluster creation on bare metal and vSphere with minimal intervention. ZTP also offers a GitOps workflow for managing deployments.
+* Support for hosted control planes for {product-title}.
```
would it be OK to include:
"(from the **HyperShift** project) somewhere on this page so that if people search for that term in the doc, it will land them here?"

julienlim replied:
Seems odd that we give a 1-liner on what Hosted control planes does (whereas the other bullets do).  Suggesting the following instead:

Hosted control planes provides a centralized control plane for OpenShift clusters which enables you to operate multiple clusters in a hyperscale manner. Hosted control planes is available on {product-title}.

berenss commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,28 @@
+If you use {product-title} 4.8.2 or later, you can separately install multicluster engine to access the following capabilities:
+
+* Hive, which provisions and manages self-managed OpenShift clusters to the hub.
+* Zero touch provisioning (ZTP), which provides full automation for cluster creation on bare metal and vSphere with minimal intervention. ZTP also offers a GitOps workflow for managing deployments.
```
this could link to the ZTP doc topic in OCP doc?

racedo replied:
+1 to add a link to the ZTP docs. If the new ZTP docs here https://docs.openshift.com/container-platform/4.11/scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.html apply to MCE this should link to them

berenss commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,28 @@
+[id="mce-additional-resources-ocp"]
+== Additional resources
+
+For the complete documentation for multicluster engine, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/multicluster_engine/index[multicluster engine documentation], which is part of the product documentation for Red Hat Advanced Cluster Management. 
```
Should we also include a note to the effect of:
"If you are entitled to Red Hat Advanced Cluster Management you can deploy the operator and get full management capabilities.  See the Red Hat Advanced Cluster Management install doc topic <link>"

berenss commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,28 @@
+* Infrastructure Operator, which manages the deployment of the Assisted Service to orchestrate on-premise bare metal and vSphere installations of OpenShift, such as SNO on bare metal. The Infrastructure Operator includes Central Infrastructure Management (CIM), which provides an interactive user interface in the Red Hat Advanced Cluster Management console where you can provision clusters.
+* Open cluster management, which provides resources to manage multiple Kubernetes clusters.
+
+If you use {product-title} 4.8.2 or later, you can separately install multicluster engine to access the following capabilities:
```
this note is in an odd location. is it out of order?

berenss reviewed (changes requested):
looking really good to me!

zanetworker commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,28 @@
+
+One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator. The operator delivers full lifecycle capabilities for managed OpenShift clusters and partial lifecycle management for other Kubernetes distributions. It is available as part of Red Hat Advanced Cluster Management fo ...
+
+If you use Red Hat Advanced Cluster Management, multicluster engine is included and provides the following capabilities:
```
```suggestion
The multicluster engine provides the following capabilities:
```

Adding "If you use RHACM ..." is confusing, brings the reader back to asking "oh I need ACM, hmmm..."

berenss replied:
agree

zanetworker commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,28 @@
+
+toc::[]
+
+One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator. The operator delivers full lifecycle capabilities for managed OpenShift clusters and partial lifecycle management for other Kubernetes distributions. It is available as part of Red Hat Advanced Cluster Management fo ...
```
```suggestion
One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator. The operator delivers full lifecycle capabilities for managed OpenShift clusters and partial lifecycle management for other Kubernetes distributions. It is available as a standalone operator that is supported as part of your OpenShift or Red Hat OpenShift Kubernetes Engine subscription or as part of Red Hat Advanced Cluster Management for Kubernetes.
```

zanetworker replied:
This is OpenShift documentation, makes sense to start with OpenShift first IMO.

julienlim commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,28 @@
+
+If you use {product-title} 4.8.2 or later, you can separately install multicluster engine to access the following capabilities:
+
+* Hive, which provisions and manages self-managed OpenShift clusters to the hub.
```
Suggest to expand to the following:
Hive, which provisions and performs initial configurations for self-managed OpenShift clusters to the hub.

lahinson commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,28 @@
+* Zero touch provisioning (ZTP), which provides full automation for cluster creation on bare metal and vSphere with minimal intervention. ZTP also offers a GitOps workflow for managing deployments.
+* Support for hosted control planes for {product-title}.
+
+The operator is entitled with a Red Hat OpenShift support subscription and is delivered separately from the core OpenShift payload. To start to use mulicluster engine, you must deploy the OpenShift cluster and then install the operator. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/multicluster_engine/multi ...
```
Good catch, Chris!

lahinson commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,28 @@
+
+toc::[]
+
+One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator. The operator delivers full lifecycle capabilities for managed OpenShift clusters and partial lifecycle management for other Kubernetes distributions. It is available as part of Red Hat Advanced Cluster Management fo ...
```
@julienlim Per the product naming guidelines, we don't use the MCE abbreviation in the product documentation.

berenss replied:
We got legal approval (earlier in this year) from RH naming and brand to use MCE after it has been introduced. Brandi / Chris can confirm.

lahinson replied:
@berenss Thanks. I'm re-confirming with Brandi. While I was working on my first draft of this a couple weeks ago, I had chatted with her about whether we use "MCE" in the docs, and she said that we don't.

lahinson replied:
Brandi just confirmed again that we don't abbreviate MCE in the documentation.

lahinson commented:
@racedo @zanetworker @julienlim @cadawson @berenss Thank you all for your review and feedback! I updated the draft per your comments. As you'll see, I restructured the content to focus more on MCE within the context of OCP. Hopefully that will resolve any confusion about whether OCP customers need RHACM. The updated preview link is in the first comment in this PR.

zanetworker commented:
@lahinson Thank you very much for working on this. 

One tiny piece that might have confused me a bit is the section "beyond OpenShift". I.e., MCE is MCE with or without ACM, do we want to highlight ACM itself in this section? @berenss is there really a difference when you run MCE entitled with MCE or not?

lahinson commented:
@zanetworker I'm glad you mentioned the "beyond" heading because I wasn't sure if it was quite right. If it makes more sense, I can update that heading to clarify that the text in that section is specifically about MCE with ACM. Before I do that, I'll wait to see if @berenss has thoughts about it.

lahinson commented:
Here's a screenshot of my local build of the topic with the heading changed to "Capabilities of multicluster engine with Red Hat Advanced Cluster Management" the first sentence in that section is also slightly different. LMK if that's better.

<img width="792" alt="Screenshot 2022-12-01 at 11 20 47 AM" src="https://user-images.githubusercontent.com/102675557/205105368-84a8ecae-0657-4158-9a50-c0d692a7e009.png">

lahinson commented:
@zanetworker @berenss Thanks for the additional feedback! I made a couple more updates to the draft, specifically to the section that focuses on RHACM. Let me know what you think when you can.

berenss commented:
Honestly I'm having a hard time to understand what our goal is of this section:
![image](https://user-images.githubusercontent.com/20711730/205372053-fae00e28-2203-4386-8645-7cda97d11a23.png)

Those 3 bullets are in the MCE. 
conflating MCE and RHACM in the title of the section 
> Capabilities of multicluster engine with Red Hat Advanced Cluster Management
has me confused.

MCE is MCE. It doesn't need RHACM to do anything.

Now if the MCE user ends up deploying a lot of clusters using MCE tech, then they might find the need to get into RHACM which builds on top of the MCE foundation. So they would deploy the RHACM operator and get voila 17 new whizbang features.

berenss commented:
Ok looking good.
As for those 3 bullets, did they get moved up into the list of items which MCE provides?

lahinson commented:
@berenss Nope, I removed those 3 items -- they were klusterlet, CIM, and open cluster management. For some reason, based on what I saw in the MCE FAQ, I thought that the main capabilities that you gain when you enable MCE on OCP are ZTP, Hive, and hosted control planes. Should I add klusterlet, CIM, and open cluster management to that list? I am learning as I go here, so I appreciate your patience and advice 😄

berenss commented:
Yes please maintain those 3 bullets and add them to the above 3. All 6 are capabilities of the MCE.

berenss commented:
any particular reason we lost the ZTP bullet in the latest commit?
* link:https://docs.openshift.com/container-platform/4.11/scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.html[Zero touch provisioning (ZTP)], which fully automates cluster creation on bare metal and vSphere provisioning with GitOps workflows to manage deployments and configuration changes.

lahinson commented:
/label peer-review-needed

bscott-rh commented:
/label peer-review-in-progress

bscott-rh commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,35 @@
+
+toc::[]
+
+One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator (MCE). The operator delivers full lifecycle capabilities for managed OpenShift clusters and partial lifecycle management for other Kubernetes distributions. It is available in two ways:
```
```suggestion
One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator (MCE). The operator delivers full lifecycle capabilities for managed {product-title} clusters and partial lifecycle management for other Kubernetes distributions. It is available in two ways:
```
Replaced "openshift" with the attribute variable.

bscott-rh commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,35 @@
+
+One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator (MCE). The operator delivers full lifecycle capabilities for managed OpenShift clusters and partial lifecycle management for other Kubernetes distributions. It is available in two ways:
+
+* As a standalone operator that you install as part of your OpenShift or Red Hat OpenShift Kubernetes Engine subscription
```
```suggestion
* As a standalone operator that you install as part of your {product-title} or {oke} subscription
```
Replaced the product names with the attribute variables.

bscott-rh commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,35 @@
+
+When you enable multicluster engine on {product-title}, you gain the following capabilities:
+
+* xref:control-plane.adoc#hosted-control-planes-overview_control-plane[Hosted control planes], which is based on the HyperShift project. With hosted control planes, you have a centralized control plane for OpenShift clusters so that you can operate clusters in a hyperscale manner.
```
I'd recommend either "Hosted control planes, which are based on..." or "Hosted control planes, which is a feature based on..." 

Also, is "hyperscale" a well known technical term? It's new to me and I'm wondering if you should either use a more generic word or define "hyperscale".  Just a couple of wording suggestions and replacing "openshift" with the variable.

```suggestion
* xref:control-plane.adoc#hosted-control-planes-overview_control-plane[Hosted control planes], which are based on the HyperShift project. Centralized hosted control planes let you operate {product-title} clusters in a hyperscale manner.
```

lahinson replied:
Good feedback here! 

I think I'll use "Hosted control planes, which is a feature that is based on the HyperShift project." This is one of those tricky feature names  because technically it's singular, but it sounds like we're talking about multiple things. 

I had the same thought about "hyperscale" being not well known enough, but I did some searching, and it looks like it's a term that's being adopted in the industry. I don't find it mentioned in the ISG, SSG, or the OpenShift glossary, but then again, those sources don't define every technical term. I'm tempted to leave it for now unless we get more feedback that it's unclear. 

I like your suggested rewrite, but I might tweak it a bit to avoid using the phrase "hosted control planes let you...". The ISG warns against phrases like "lets you..." or "allows you..." because by using those phrases, we're assigning human characteristics to inanimate objects (see [Anthropomorphism](https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism)). 

I'm thinking of changing it to something like, "Hosted control planes, which is a feature that is based on the HyperShift project. With a centralized hosted control plane, you can operate {product-title} clusters in a hyperscale manner."

bscott-rh replied:
This sounds great to me!

bscott-rh commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,35 @@
+When you enable multicluster engine on {product-title}, you gain the following capabilities:
+
+* xref:control-plane.adoc#hosted-control-planes-overview_control-plane[Hosted control planes], which is based on the HyperShift project. With hosted control planes, you have a centralized control plane for OpenShift clusters so that you can operate clusters in a hyperscale manner.
+* Hive, which provisions self-managed OpenShift clusters to the hub and completes the initial configurations for those clusters.
```
```suggestion
* Hive, which provisions self-managed {product-title} clusters to the hub and completes the initial configurations for those clusters.
```

I'm not familiar with this area of the documentation - is "the hub" a well known term here or should it be described more specifically here?

lahinson replied:
I believe "the hub" will be understood by customers who are familiar with cluster management.

bscott-rh commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,35 @@
+* xref:control-plane.adoc#hosted-control-planes-overview_control-plane[Hosted control planes], which is based on the HyperShift project. With hosted control planes, you have a centralized control plane for OpenShift clusters so that you can operate clusters in a hyperscale manner.
+* Hive, which provisions self-managed OpenShift clusters to the hub and completes the initial configurations for those clusters.
+* klusterlet agent, which registers managed clusters to the hub.
+* Infrastructure Operator, which manages the deployment of the Assisted Service to orchestrate on-premise bare metal and vSphere installations of OpenShift, such as SNO on bare metal. The Infrastructure Operator includes link:https://docs.openshift.com/container-platform/4.11/scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.html[zero touch provisioning (ZTP)], whi ...
```
```suggestion
* Infrastructure Operator, which manages the deployment of the Assisted Service to orchestrate on-premise bare metal and vSphere installations of {product-title}, such as SNO on bare metal. The Infrastructure Operator includes link:https://docs.openshift.com/container-platform/4.11/scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.html[zero touch provisioning (ZTP)], which fully automates cluster creation on bare metal and vSphere provisioning with GitOps workflows to manage deployments and configuration changes.
```

Should this link be an xref?

lahinson replied:
@bscott-rh  I had the same question, and I initially tried to make it an xref, but I couldn't get it to work. Then, I noticed that I couldn't find that file anywhere in the openshift-docs repo, so I gave up and used a link. I can play around with it again this morning to see if I can get an xref to work.

bscott-rh commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,35 @@
+* Infrastructure Operator, which manages the deployment of the Assisted Service to orchestrate on-premise bare metal and vSphere installations of OpenShift, such as SNO on bare metal. The Infrastructure Operator includes link:https://docs.openshift.com/container-platform/4.11/scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.html[zero touch provisioning (ZTP)], whi ...
+* Open cluster management, which provides resources to manage Kubernetes clusters.
+
+The multicluster engine is entitled with your Red Hat OpenShift support subscription and is delivered separately from the core OpenShift payload. To start to use multicluster engine, you deploy the OpenShift cluster and then install the operator. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/multicluster_en ...
```
```suggestion
The multicluster engine is included? with your Red Hat {product-title} support subscription and is delivered separately from the core {product-title} payload. To use multicluster engine, you deploy the {product-title} cluster and then install the operator. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/multicluster_engine/multicluster_engine_overview#mce-install-intro[the installation documentation].
```
Brief wording suggestion.

bscott-rh commented:
Looks good! I left a few wording suggestions and noted a few places where you could consider replacing "OpenShift" with the attribute variable {product-title}. Let me know what you think!

lahinson commented:
@bscott-rh Thanks so much for your feedback! I updated the draft per your suggestions and added a couple of responses to your questions. One question: I updated `Red Hat OpenShift Kubernetes Engine` to `{oke}`, but it isn't rendering:

![Screenshot 2022-12-06 at 11 01 14 AM](https://user-images.githubusercontent.com/102675557/205961834-2deb4f06-49da-43b3-8efe-0245b75473db.png)

Do I need to define something else in the file to make that attribute render correctly?

bscott-rh commented:
LGTM. 

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

lahinson commented:
/label merge-review-needed

lahinson commented:
/label merge-review-needed

lahinson commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,35 @@
+[id="mce-on-rhacm"]
+== Cluster management with Red Hat Advanced Cluster Management
+
+If you need cluster management capabilities beyond what {product-title} with multicluster engine can provide, consider Red Hat Advanced Cluster Management. The multicluster engine is an integral part of Red Hat Cluster Management and is enabled by default. 
```
Good catch!

lahinson replied:
Fixed

dtthuynh commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,35 @@
+* Infrastructure Operator, which manages the deployment of the Assisted Service to orchestrate on-premise bare metal and vSphere installations of {product-title}, such as SNO on bare metal. The Infrastructure Operator includes xref:../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.adoc#ztp-challenges-of-far-edge-deployments_ztp-deploying-far-edge-clusters-at-sca ...
+* Open cluster management, which provides resources to manage Kubernetes clusters.
+
+The multicluster engine is included with your {product-title} support subscription and is delivered separately from the core payload. To start to use multicluster engine, you deploy the {product-title} cluster and then install the operator. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/multicluster_engine/m ...
```
will we be refreshing this to the latest doc when 2.7+ comes out?

lahinson replied:
Yes. The OCP docs go live about 2 weeks before ACM 2.7 is released, so I have a to-do for myself to update this link after ACM 2.7 is live.

dtthuynh commented on architecture/mce-overview-ocp.adoc:
```diff
@@ -0,0 +1,35 @@
+[id="mce-on-rhacm"]
+== Cluster management with Red Hat Advanced Cluster Management
+
+If you need cluster management capabilities beyond what {product-title} with multicluster engine can provide, consider Red Hat Advanced Cluster Management. The multicluster engine is an integral part of Red Hat Advanced Cluster Management and is enabled by default. 
```
should we link to the ACM landing page or doc here?

lahinson replied:
I linked to the ACM landing page at the first mention of ACM near the beginning of the topic.

dtthuynh commented:
Overall looks good to me. Thanks @lahinson

lahinson commented:
Thanks so much for taking a look, @dtthuynh! I really appreciate it.

lahinson commented:
/label merge-review-needed

mjpytlak commented:
/cherrypick enterprise-4.12

lahinson commented:
🎉 🚀 😅
